doc/whats_new/v0.20.rst

```diff
@@ -38,6 +38,10 @@ Changelog
   even if all transformation results are sparse. :issue:`12304` by `Andreas
   Müller`_.
 
+- |API| :class:`compose.ColumnTransformer` now expects
+  `(name, columns, transformer)` instead of `(name, transformer, columns)`.
+  :issue:`12339` by :user:`Adrin Jalali <adrinjalali>`.
```

Comment:
Perhaps note that a warning will be raised until version x if the 0.20.0 order is used.